testops-ldap/docker-compose.yml

```diff
@@ -96,12 +103,15 @@ services:
       SPRING_DATASOURCE_URL: jdbc:postgresql://${ALLURE_UAA_DB_HOST}:${ALLURE_UAA_DB_PORT}/${ALLURE_UAA_DB_NAME}?assumeMinServerVersion=11&ApplicationName=${ALLURE_UAA_SERVICE_NAME}
       SPRING_DATASOURCE_USERNAME: ${ALLURE_UAA_DB_USERNAME}
       SPRING_DATASOURCE_PASSWORD: ${ALLURE_UAA_DB_PASS}
+      SPRING_DATASOURCE_HIKARI_MAXIMUMPOOLSIZE: ${ALLURE_UAA_DB_MAXIMUMPOOLSIZE}
+      MANAGEMENT_METRICS_EXPORT_PROMETHEUS_ENABLED: 'false'
+      MANAGEMENT_HEALTH_DISKSPACE_ENABLED: 'false'
       MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE: health,info,prometheus,configprops
       MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIME-TO-LIVE: 15s
       SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI: http://allure-gateway:${ALLURE_GATEWAY_PORT}/.well-known/jwks.json
       ALLURE_LOGIN_PRIMARY: ldap
       ALLURE_LOGIN_LDAP_ENABLED: 'true'
-      ALLURE_LOGIN_LDAP_REFERAL: follow
+      ALLURE_LOGIN_LDAP_REFERRAL: follow
```

Comment:
<img width="1242" alt="image" src="https://github.com/qameta/allure-docker-compose/assets/8607348/0a62c5b7-46aa-4456-9f95-e7fb7f3fbbb0">
in the code we have RR here `REFERRAL`
@alexey-bogdanov